### PR TITLE
Fixes on registration list

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: indico/MaKaC/PDFinterface/conference.py [hunk 1]
```diff
@@ -1614,6 +1614,10 @@ def getBody(self, story=None, indexedFlowable={}, level=1 ):
             lp.append(Paragraph('<b>{}</b>'.format(_('Registration state')), text_format))
         if 'price' in self.special_items:
             lp.append(Paragraph('<b>{}</b>'.format(_('Price')), text_format))
+        if 'checked_in' in self.special_items:
+            lp.append(Paragraph('<b>{}</b>'.format(_('Checked in')), text_format))
+        if 'checked_in_date' in self.special_items:
+            lp.append(Paragraph('<b>{}</b>'.format(_('Checked in date')), text_format))
```
Comment (ThiefMaster) [inline indico/MaKaC/PDFinterface/conference.py]: `Check-in date`?

File: indico/modules/events/registration/controllers/management/reglists.py [hunk 2]
```diff
@@ -71,8 +71,9 @@
         'id': 'price'
     },
     'state': {
-        'title': _('Status'),
-        'id': 'state'
+        'title': _('State'),
+        'id': 'state',
+        'filter_choices': {str(state.value): state.title for state in RegistrationState.__members__.itervalues()}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/controllers/management/reglists.py]: You can iterate over `RegistrationState` directly

File: indico/MaKaC/PDFinterface/conference.py (hunk 1 above)
Comment (nop33) [inline indico/MaKaC/PDFinterface/conference.py]: :+1:

File: indico/MaKaC/PDFinterface/conference.py [hunk 3]
```diff
@@ -1476,9 +1476,11 @@ def _print_row(caption, value):
 
         for item in self._display:
             if item.input_type == 'accommodation' and item.id in data:
-                _print_row(caption=item.title, value=data[item.id].friendly_data['choice'])
-                _print_row(caption=_('Arrival date'), value=format_date(data[item.id].friendly_data['arrival_date']))
-                _print_row(caption=_('Departure date'), value=format_date(data[item.id].friendly_data['departure_date']))
+                _print_row(caption=item.title, value=data[item.id].friendly_data.get('choice'))
+                _print_row(caption=_('Arrival date'),
+                           value=format_date(data[item.id].friendly_data.get('arrival_date')))
```
Comment (ThiefMaster) [inline indico/MaKaC/PDFinterface/conference.py]: Won't this fail if there's no date?

**Pull request closed without merging**